clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerProtocol.java

```diff
@@ -30,24 +30,32 @@
 import java.util.List;
 import java.util.Map;
 
+import static org.apache.kafka.common.protocol.CommonFields.ERROR_CODE;
```

Comment:
I think it's slightly dubious to use the same error code field from the common protocol. It just saves us one field definition and the doc string doesn't make as much sense. Also, we'll probably be getting rid of `CommonFields` eventually.